docs/docs/getting-started/componentDefinition.mdx

```diff
@@ -0,0 +1,13 @@
+A component is a Helm Chart that can be installed on an Airy Cluster in order to extend its functionality.
```

Comment:
We do call components also the components which are part of `Airy Core`: `api-admin`, `api-communication` :thinking: Perhaps a collection of functionalities and resources? There some components which are part of Core and the others can be installed on top.